api/utils/endpoints.go

```diff
@@ -1,57 +1,60 @@
 package utils
 
 import (
-	"fmt"
 	"log"
+	"net"
 	"net/url"
-	"path"
 	"strconv"
+	"strings"
 )
 
+const apiPrefix = "/api"
+
 func ApiListenUrl() *url.URL {
+	const defaultIP = "127.0.0.1"
```

Comment:
Could you group those const values together? Also `apiPrefix` is `defaultAPIPrefix` actually.